### PR TITLE
Remove trailing commas

### PR DESCRIPTION
**Pull request by derekwaynecarr**
Fixes https://github.com/GoogleCloudPlatform/kubernetes/issues/12446

### CODE REVIEW

File: docs/design/namespaces.md [hunk 1]
```diff
@@ -267,13 +267,13 @@ OpenShift creates a Namespace in Kubernetes
...
   "spec": {
-    "finalizers": ["openshift.com/origin", "kubernetes"],
+    "finalizers": ["openshift.com/origin", "kubernetes"]
   },
   "status": {
-    "phase": "Active",
+    "phase": "Active"
   },
   "labels": {
     "name": "development"
```
Comment (cjcullen) [inline docs/design/namespaces.md]: I think we need to kill off the commas after "labels" too, right? (in all three places)

**Comment by cjcullen** [comment]
Ping @derekwaynecarr

**Comment by derekwaynecarr** [comment]
@cjcullen - I was on vacation.  I will update this PR now and poke when ready for review.

**Comment by derekwaynecarr** [comment]
@cjcullen - updated doc and validated that I was able to create the proper namespace manually.  had to move labels into metadata as well.

**Comment by cjcullen** [comment]
LGTM

**Pull request merged**